doc/requirements/srsd.rst

```diff
@@ -528,7 +528,7 @@ TODO: This is completed by the :doc:`Settings Report </user/inputs/settings_repo
    :id: REQ_REACTOR_CORRECTNESS
    :status: needs implementation, needs test
 
-The reactor package shall check its inputs for certain obvious errors including unphysical quantities. At a deep level, the reactor package will not attempt to full validate subtle engineering aspects of the reactor; that is more generally the reason users will want to fully simulate a reactor and cannot be done at input time.
+The reactor package shall check its inputs for certain obvious errors including unphysical quantities. At a deep level, the reactor package will not attempt to fully validate subtle engineering aspects of the reactor; that is more generally the reason users will want to fully simulate a reactor and cannot be done at input time.
```

Comment:
This requirement should be broken down into specifics so that it is actually verifiable

Reply:
I have seriously reworded this requirement (and shortened the language).  We already have a bunch of other requirements that specify HOW we validate validate settings: min/max bounds, names, types, defaults, and custom rules.

Looking at this requirement... I'm not sure it's actionable to say we test for "unphysical input data" other than the settings validation we already have requirements for:

* min/max bounds
* default
* (optionally) custom rules

I think the only two options that make sense to me here are:

1. make several of the other, existing, settings requirements a sub-requirement of this one
2. OR remove this one, as it does not contain anything useful in itself

Reply:
To be honest, I'm not sure we DO much checking for physical validity of the reactor inputs. 

@ntouran How much blueprint / reactor validation do we really do?  I ask because, at some level, we won't be able to validate correctness while reading in the blueprints, and will have to wait for failure during runtime.  I'm thinking of the [haulting problem](https://en.wikipedia.org/wiki/Halting_problem).

Reply:
I'm good with what we have as an great template to build on. We don't need all the requirements in right now because it will take time to develop. We should balance requirements around what we currently have versus what we will want to have. Additionally, it's just important that requirements are broken down to have a simple component that can be verifiable such that if we verify all child requirements under some parent we satisfy the broader parent (if that's how the requirements hierarchy is developed)

Reply:
This is definitely only the first serious pass at requirements for ARMI.  I don't want anything faulty in here, but this is also a big enough lift that I expect it to be on-going work for a while.

I have my next steps laid out, though perhaps I should open a "Discussion" for them, to make that process more visible to the team.

Thanks!